tests/infra/network.py

```diff
@@ -540,7 +539,7 @@ def _setup_common_folder(self, constitution):
             ), f"Could not copy governance {fragment} to {self.common_dir}"
         # It is more convenient to create a symlink in the common directory than generate
         # certs and keys in the top directory and move them across
-        cmd = ["cp"] if IS_SNP else ["ln", "-s"]
+        cmd = ["ln", "-s"]
```

Comment:
Taking a punt here, assuming it's now safe to symlink everywhere? If I'm missing something, we can restore this branch based on SNP-support, or something more precise.

Reply:
yes